packages/terra-signature/tests/wdio/signature-spec.js

```diff
@@ -0,0 +1,64 @@
+/* global browser, Terra */
+
+const viewports = Terra.viewports('medium');
+
+describe('Signature', () => {
+  describe('Default', () => {
```

Comment:
Looks like you checked in a 'large' or 'huge' viewport sized screenshot for 'Default' but is not longer defined in the test file

Reply:
It's removed here. https://github.com/cerner/terra-core/pull/1558/commits/56ac2511e13e96325b4308e03156c921bf564617